tests/test_context.py

```diff
@@ -471,27 +417,12 @@ def setUpClass(cls):
         super().setUpClass()
         component.registry.register(name="simple_component", component=SimpleComponent)
 
-    @override_settings(
-        COMPONENTS={"context_behavior": "django"},
-    )
-    def test_simple_component_can_use_outer_context__django(self):
-        template_str: types.django_html = """
-            {% load component_tags %}{% component_dependencies %}
-            {% component 'simple_component' %}{% endcomponent %}
-        """
-        template = Template(template_str)
-        rendered = template.render(Context({"variable": "outer_value"}))
-        self.assertHTMLEqual(
-            rendered,
-            """
-            Variable: <strong> outer_value </strong>
-            """,
-        )
-
-    @override_settings(
-        COMPONENTS={"context_behavior": "isolated"},
-    )
-    def test_simple_component_cannot_use_outer_context__isolated(self):
+    # NOTE: Second arg in tuple is expected value.
+    @parametrize_context_behavior([
+        ("django", "outer_value"),
+        ("isolated", ""),
+    ])
+    def test_simple_component_can_use_outer_context(self, context_behavior_data):
```

Comment:
Here's an example of how we can pass extra data along with the modes in `parametrize_context_behavior`.